docker/1.8.0/final/py2/Dockerfile.cpu

```diff
@@ -7,20 +7,22 @@ ARG framework_support_installable=sagemaker_tensorflow_container-1.0.0.tar.gz
 
 WORKDIR /root
 
+COPY $framework_installable .
 COPY $framework_support_installable .
 
-RUN framework_support_installable_local=$(basename $framework_support_installable) && \
+RUN framework_installable_local=$(basename $framework_installable) && \
```

Comment:
What's the difference between a framework_installable and a framework_support_installable?

Reply:
framework_installable is tensorflow 
framework_support is sagemaker-tensorflow-container